app/models/form.rb

```diff
@@ -12,11 +12,11 @@ def initialize(form_path)
   end
 
   def get_subsection(id)
-    subsections.find { |s| s.id == id }
+    subsections.find { |s| s.id == id.to_s.underscore }
```

Comment:
Might be better to just expect them to be dasherized in the form definition JSON? We do control that spec after all.

Reply:
I tried that approach initially, but that involved changing more areas of the codebase (i.e. things like `question.hint_text` would need to be `question["hint-text"]`). But am happy with either approach.

Reply:
Wouldn't have to be consistent for all keys in the JSON, just page ids (although I appreciate that makes a find and replace fiddly). Not particularly important I think this is fine to go in as is.